specification/sql/resource-manager/Microsoft.Sql/preview/2023-02-01-preview/LocationCapabilities.json

```diff
@@ -420,6 +420,40 @@
         }
       }
     },
+    "FreeLimitExhaustionBehaviorCapability": {
+      "description": "Supported free limit exhaustion behavior options",
+      "type": "object",
+      "properties": {
+        "name": {
+          "description": "Free limit exhaustion behavior name",
+          "enum": [
+            "AutoPause",
+            "BillOverUsage"
+          ],
+          "type": "string",
+          "readOnly": true,
+          "x-ms-enum": {
+            "name": "FreeLimitExhaustionBehavior",
+            "modelAsString": true
+          }
+        },
+        "status": {
+          "description": "Free limit exhaustion behavior status",
+          "enum": [
+            "Visible",
+            "Available",
+            "Default",
+            "Disabled"
+          ],
+          "type": "string",
+          "readOnly": true,
+          "x-ms-enum": {
+            "name": "CapabilityStatus",
+            "modelAsString": false
```

Comment:
>"modelAsString": false [](http://example.com/codeflow?start=12&length=22)

Generally recommend true for this. Any reason you chose false?

Reply:
only accepted for this PR as other "status" in this doc also defined "false"